extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -473,12 +473,12 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
 
       DicomMetadataStore.addSeriesMetadata(seriesSummaryMetadata, madeInClient);
 
-      const numberOfSeries = seriesPromises.length;
```

Comment:
The code here returns before the series actually get loaded, breaking the contract on the call.  That made it impossible to know when the series load finished, which is the point required to actually determine the hanging protocol.